### PR TITLE
Update 3-1-calculator.md

### PR DESCRIPTION
**Pull request by Alex-Wu-AU**
Fixed the errors in calculation and removed the redundant 4th loop.

### CODE REVIEW

File: src/content/docs/book/part-1-instructions/3-control-flow/3-explore/3-1-calculator.md [hunk 1]
```diff
@@ -19,13 +19,13 @@ Average: 10.0
 Add another value: [y/n] y
 Enter value: 30
 Count: 2
-Total: 30.0
+Total: 40.0
 Min: 10.0
 Max: 30.0
 Average: 15.0
```
Comment (coskun-kilinc) [inline src/content/docs/book/part-1-instructions/3-control-flow/3-explore/3-1-calculator.md:25]: I think average should be 20 here? 
$\frac{\textrm{total}}{\textrm{count}}\equiv\frac{40}{2}=20$

File: src/content/docs/book/part-1-instructions/3-control-flow/3-explore/3-1-calculator.md [hunk 2]
```diff
@@ -40,13 +40,6 @@ Min: -45.0
 Max: 30.0
 Average: 0.0
 
-Add another value: [y/n] y
```
Comment (coskun-kilinc) [inline src/content/docs/book/part-1-instructions/3-control-flow/3-explore/3-1-calculator.md:43]: Good catch removing the duplicate loop.

**Review by coskun-kilinc — CHANGES_REQUESTED** [review]
Great changes, I just think (correct me if I'm wrong!) we need to update the average in the second loop. 😊

File: src/content/docs/book/part-1-instructions/3-control-flow/3-explore/3-1-calculator.md (hunk 1 above)
Comment (Alex-Wu-AU) [reply src/content/docs/book/part-1-instructions/3-control-flow/3-explore/3-1-calculator.md:25]: Yes that's correct! I missed that

**Review by coskun-kilinc — APPROVED** [review]
Awesome!

**Review by Alex-Wu-AU — COMMENTED** [review]
Correct, average should be 20.

**Review by coskun-kilinc — APPROVED** [review]

**Pull request merged**